misc.go

```diff
@@ -53,6 +53,19 @@ func runeSliceWidthRange(rs []rune, beg, end int) []rune {
 	return rs[b:]
 }
 
+// Returns the last runes of `rs` that take up at most `max_width` space.
+func runeSliceWidthLastRange(rs []rune, max_width int) []rune {
+	lastWidth := 0
+	for i := len(rs) - 1; i >= 0; i-- {
+		w := runewidth.RuneWidth(rs[i])
+		if lastWidth+w > max_width {
```

Comment:
This should be camelcase too, apologies I missed it last time.

```suggestion
		if lastWidth+w > maxWidth {
```